### PR TITLE
Release automation

### PR DESCRIPTION
**Pull request by JamesLindsay0**
## Changeset

- `rake "plugin:bump[1.2.3]"` to update the build.sh and changelog
- `rake plugin:release` to add the tag
- `rake plugin:package` to build, test, and upload the package

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -152,7 +264,7 @@ namespace :plugin do
       task all_android64: [:assets, :csharp]
     else
       task all: [:assets, :cocoa, :android, :csharp, ]
-      task all_android64: [:assets, :cocoa, :android_64bit, :csharp ]
```
Comment (richardelms) [inline Rakefile:155]: Better get this back in

File: Rakefile [hunk 2]
```diff
@@ -162,16 +274,7 @@ namespace :plugin do
       sh "git", "clean", "-dfx", "unity"
       # remove cocoa build area
       FileUtils.rm_rf cocoa_build_dir
-      unless is_windows?
```
Comment (richardelms) [inline Rakefile:165]: Is this left from trying to fix android builds?

**Review by richardelms — CHANGES_REQUESTED** [review]
Couple things missing by mistake i think

File: Rakefile (hunk 2 above)
Comment (JamesLindsay0) [reply Rakefile:165]: Yes - I've reverted the change now. Thanks for spotting.

**Review by richardelms — APPROVED** [review]
Thanks!

**Pull request merged**